lisp/sns.lisp

```diff
@@ -245,3 +245,16 @@
                                               (aref elt (1+ i))))
                              (format nil "~A/~D.dat" output-directory i)
                              :if-exists if-exists))))
+
+(defun send-list (chan data)
```

Comment:
At least name differently.  Maybe can make more general, e.g., Map a constructor over a list of messages.